test/e2e/BUILD

```diff
@@ -210,20 +210,13 @@ go_library(
     ],
 )
 
-go_test(
+# phony go_default_test. Bazel test run interprets these as
```

Comment:
I can't fix this without modifying rules_go. I will add it to the list of stuff to finish.